emscripten.py

```diff
@@ -753,7 +753,8 @@ def create_receiving(exports):
       for s in exports_that_are_not_initializers:
         mangled = asmjs_mangle(s)
         dynCallAssignment = ('dynCalls["' + s.replace('dynCall_', '') + '"] = ') if generate_dyncall_assignment and mangled.startswith('dynCall_') else ''
-        receiving += [dynCallAssignment + mangled + ' = asm["' + s + '"];']
+        export_assignment = 'Module["{}"] = asm["{}"];'.format(mangled, s) if settings.MODULARIZE and settings.EXPORT_ALL else 'asm["{}"]'.format(s)
+        receiving += [dynCallAssignment + mangled + ' = {};'.format(export_assignment)]
```

Comment:
Hmm, is there a specific reason to have the `asm['x']` part be formatted on both sides of the branch?

Now that we are fully Python 3, I would recommend using f-strings over `.format()` and `%`, e.g.

```
export_assignment = f'Module["{mangled}"] =' if settings.MODULARIZE and settings.EXPORT_ALL else ''
receiving += [f'{export_assignment} {dynCallAssignment} {mangled} = asm["{s}"]']
```
might read the cleanest without duplication?

Reply:
Done, use fstrings